lib/maintainers.nix

```diff
@@ -95,4 +95,10 @@
     githubId = 30784060;
     name = "Pedro Sánchez";
   };
+  insipx = {
+    email = "github.tech@liquidthink.net";
+    github = "insipx";
+    githubId = 6452260;
+    name = "Andrew Plaza";
+  };
```

Comment:
Can you do this in a separate commit, `"maintainers: add insipx"`?

This commit should come before the main one, which will be titled `"plugins/scope: init"`.